packages/communication-react/review/communication-react.api.md

```diff
@@ -829,6 +865,9 @@ export interface IncomingCallState {
     startTime: Date;
 }
 
+// @public
+export const isChatErrorTarget: (target: string) => boolean;
```

Comment:
Can this be ommitted from the public api or will external devs need to make use of it also?

Reply:
+1 I thought of this later last night. I'll move the implementation to the adapter that uses it.

Reply:
Done.